hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/expiry/ExpiryProcessTest.java

```diff
@@ -132,7 +132,7 @@ class ExpiryProcessTest {
     @BeforeEach
     void setUp() {
         setUpPreRequisites();
-        subject = new ExpiryProcess(classifier, renewalWork, removalWork, consensusTimeTracker);
+        subject = new ExpiryProcess(classifier, renewalWork, removalWork, dynamicProperties, consensusTimeTracker);
```

Comment:
You can't write this as a unit test because it involves multiple classes wired up by Dagger.  It's possible (I think) to write it as a bdd test.  But ... beyond my capability now.  Investigating.